src/components/CloseVaultDialog.tsx

```diff
@@ -0,0 +1,150 @@
+import { signerTarget } from 'config';
+import { useSetAtom, useAtomValue } from 'jotai';
+import { useMemo, useState } from 'react';
+import { makeCloseVaultOffer } from 'service/vaults';
+import { displayFunctionsAtom, walletUiHrefAtom } from 'store/app';
+import { ViewMode, viewModeAtom } from 'store/vaults';
+import { motion } from 'framer-motion';
+import BaseDialog from './BaseDialog';
+import type { Amount } from '@agoric/ertp/src/types';
+
+type TableRowProps = {
+  left: string;
+  right: string;
+};
+
+const TableRow = ({ left, right }: TableRowProps) => (
+  <tr className="leading-7">
+    <td className="text-left pr-8">{left}</td>
+    <td className="text-right font-black">{right}</td>
+  </tr>
+);
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  vaultOfferId?: string;
+  totalCollateral?: Amount<'nat'>;
+  totalDebt?: Amount<'nat'>;
+};
+
+const noticeProps = {
+  initial: { height: 0, opacity: 0 },
+  animate: { height: 'auto', opacity: 1 },
+  transition: { type: 'tween' },
+};
+
+const message =
+  'Closing a vault is permanent and cannot be undone. To close your vault, please approve the offer in your wallet.';
```

Comment:
"please" seems superfluous. It won't close unless they approve. This is something they're trying to accomplish so no need to request they do something
```suggestion
  'Closing a vault is permanent and cannot be undone. To close your vault, approve the offer in your wallet.';
```